src/ui/profile_preferences.ui

```diff
@@ -134,6 +134,28 @@
         </widget>
        </item>
        <item row="1" column="0">
+        <widget class="QGroupBox" name="groupBox_encoding">
+         <property name="enabled">
+          <bool>true</bool>
+         </property>
+         <property name="title">
+          <string>Server Data Encoding</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout_7">
+          <item>
+           <widget class="QComboBox" name="comboBox_encoding">
+            <property name="enabled">
+             <bool>true</bool>
+            </property>
+            <property name="toolTip">
```

Comment:
Thinking this through, we'd like Mudlet to work straight out of the box, just install and play - and the user will only ever need to use this option if the encoding is not set correctly already. So this really is a fix, a workaround, for when Mudlet didn't do its job right. So now we have an unhappy user because text isn't displayed right and we'd like to minimise the amount of time they're unhappy, so they can enjoy Mudlet and get into the game ASAP.

In order to do that, we need to make the text as short as possible so they've got the least amount of reading to do, and simple as possible so they have the best chance of understanding it. Otherwise, we'll just be keeping them in the unhappy state longer if they have to do a lot of reading and if they don't understand what's being said, they'll definitely be even more unhappy.

This is the reasoning behind not giving as much information about the general topic as you do, but instead being super concise and using simple language to get straight to the point.

I got distracted [adding in this](http://wiki.mudlet.org/w/Manual:Geyser#Background_for_Mudlets_buttons) to the wiki so I'll provide the revised text later, but thought I'd provide the reasoning first so you'd know where I'm coming from.